app/services/arcgis_api/geocoder.rb

```diff
@@ -126,6 +126,8 @@ def handle_api_errors(response_body)
       if response_body['error']
         error_code = response_body.dig('error', 'code')
 
+        retrieve_token! if response_body['error']['code'] === 498
```

Comment:
Should we use the variable assigned on the preceding line?

```suggestion
        error_code = response_body.dig('error', 'code')

        retrieve_token! if error_code === 498
```